### ISSUE TITLE
Issues on Unity 2020.1

### ISSUE THREAD

**Issue by Haapavuo** [YarnSpinnerTool/YarnSpinner-Unity#19]
**What is the current behavior?**
Assembly 'Assets/YarnSpinner/Runtime/DLLs/System.Memory.dll' will not be loaded due to errors:
System.Memory references strong named System.Runtime.CompilerServices.Unsafe, versions has to match. Assembly references: 4.0.4.1 Found in project: 4.0.4.0.

**Please provide the steps to reproduce, and if possible a minimal demo of the problem**:
Update the System.Memory and other libraries or provide source code that is not precompiled.

**What is the expected behavior?**
Working software.

**Please tell us about your environment:**
  
  - Yarn Spinner Version: 1.2.6
  - Unity Version: 2020.1.0b4

**Comment by desplesda** [comment]
Thanks for the report - investigating.

**Comment by desplesda** [comment]
I just tested both v1.2.6 and the `develop` branch in Unity 2020.1.0b14, and I'm not seeing the error you reported, or any immediately obvious issues in any of the currently existing samples.

You reported this issue in 2020.1.0b4; does the issue still appear in 2020.1.0b14?

**Comment by Haapavuo** [comment]
Yes. I have not yet found a solution to this but I have investigated it for a while and figured out that Unity Collections package (in Package Manager) contains System.Runtime.CompilerServices.Unsafe 4.0.4.0 which conflicts with the 4.0.4.1 version that is included with Google.Protobuf. A workaround to use YarnSpinner + Unity Collections at the same time is to replace the Unsafe library inside Library/PackageCache directory with the 4.0.4.1 version (you can download the DLL from Nuget website). This is ridiculous and it is a fault inside the Unity Collections package. Unity has said that the DLL will be removed from the package in 0.11.0 version (current version is 0.9.0 so I guess it's going to take a while).

TL;DR: I close this issue since it is not totally related to YarnSpinner and cannot be fixed by YarnSpinner. Unity will fix this eventually by updating the Unity Collections package.

**Comment by desplesda** [comment]
Good to know, and thank you for the detailed followup!
